lib/improver/cli/orographic_enhancement.py

```diff
@@ -161,18 +97,34 @@ def process(temperature, humidity, pressure, wind_speed, wind_dir, orography):
         orography (iris.cube.Cube):
             Cube containing height of orography above sea level on high
             resolution (1 km) UKPP domain grid.
+        boundary_height (float):
+            Model height level to extract variables for calculating orographic
+            enhancement, as proxy for the boundary layer.
+        boundary_height_units (str):
+            Units of the boundary height specified for extracting model levels.
 
     Returns:
         iris.cube.Cube:
             Precipitation enhancement due to orography on the high resolution
             input orography grid.
     """
+    from improver.orographic_enhancement import OrographicEnhancement
+    from improver.wind_calculations.wind_components import \
+        ResolveWindComponents
+
+    constraint_info = (boundary_height, boundary_height_units)
+
+    # pylint: disable=W1637
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Id 'W1637' is used to disable 'zip-builtin-not-iterating' message emission](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4659394)